source/includes/_plaid_accounts.md

```diff
@@ -2,24 +2,25 @@
-| last_import                  | string | Timestamp in ISO 8601 extended format of the last time Lunch Money imported new data from Plaid for this account.                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| last_fetch                   | string | Timestamp in ISO 8601 extended format of the last successful check from Lunch Money for updated data or timestamps from Plaid in ISO 8601 extended format (not necessarily date of last successful import)                                                                                                                                                                                                                                                                                                                                             |
-| plaid_last_successful_update | string | Timestamp in ISO 8601 extended format of the last time Plaid successfully connected with institution for new transaction updates, regardless of whether any new data was available in the update.                                                                                                                                                                                                                                                                                                                                                      |
+Attribute Name                | Type   | Nullable | Description
+----------------------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------------------
+id                            | number | false    | Unique identifier of Plaid account
+date_linked                   | string | false    | Date account was first linked in ISO 8601 format
+name                          | string | false    | Name of the account. Can be overridden by the user. Field is originally set by Plaid
+display_name                  | string | false    | Display name of the account, if not set it will return an empty string.
```

Comment:
`display_name` should return the user defined custom name or concatenated string of institution + account name. Point being that it should always return something and the user can exclusively use this field to identify an account by a name.

Similar idea to `display_name` and `display_notes` for transactions. 

Next release will have this change!

Reply:
Perfect! I've updated the description